coderunners/services.py

```diff
@@ -80,7 +82,7 @@ def check_code(code: dict[str, str], language: str, memory_limit: int, time_limi
 
     # Process all tests
     test_results, test_scores = [], []
-    for test in test_cases:
+    for i, test in enumerate(test_cases):
```

Comment:
Let's add a print for each test case that prints the test number and the status. That would help in debugging issues in prod later. Otherwise `i` is never used.